spec/support/request_helpers.rb

```diff
@@ -267,7 +267,8 @@ def stub_wiki_validation
       'en.wikinews.org',
       'pl.wikiquote.org',
       'de.wikiversity.org',
-      'commons.wikimedia.org'
+      'commons.wikimedia.org',
+      'en.wikipedia.org'
```

Comment:
This should not be necessary, as en.wikipedia.org is the one wiki that was already present in the fixtures, so nothing in the test suite should attempt to validate it.

Reply:
It is needed as ensure_wiki_exists calls wiki_api which sends a query, which is stubbed by stub_wiki_validation and hence en.wikipedia.org needs to exist there, else VCR error comes up.

Reply:
That doesn't sound right. `ensure_wiki_exists` only gets called if you try to find a wiki that isn't already in the DB, but the fixture file gets loaded directly when the test environment boots, and bypasses that for the wikis in the fixture file.